### PR TITLE
[4.0] Fixed Language label: COM_USER_ to COM_USERS_

### PR DESCRIPTION
**Pull request by pe7er**
Some language labels had references to COM_USER (without the S).
There's no component called COM_USER; it's name is COM_USERS.

This PR replaces COM_USER_ with COM_USERS_ in the en-GB language files 
and references in PHP + XML files.

### CODE REVIEW

File: administrator/language/en-GB/en-GB.com_users.sys.ini [hunk 1]
```diff
@@ -26,25 +26,25 @@ COM_USERS_USERS_VIEW_DEFAULT_DESC="Shows a List of Users"
...
-COM_USER_REGISTRATION_VIEW_DEFAULT_DESC="Displays a registration form."
-COM_USER_REGISTRATION_VIEW_DEFAULT_OPTION="Default"
-COM_USER_REGISTRATION_VIEW_DEFAULT_TITLE="Registration Form"
-COM_USER_REMIND_VIEW_DEFAULT_DESC="Displays a username reminder request."
-COM_USER_REMIND_VIEW_DEFAULT_OPTION="Default"
-COM_USER_REMIND_VIEW_DEFAULT_TITLE="Username Reminder Request"
-COM_USER_RESET_VIEW_DEFAULT_DESC="Displays a request to reset password."
-COM_USER_RESET_VIEW_DEFAULT_OPTION="Default"
-COM_USER_RESET_VIEW_DEFAULT_TITLE="Password Reset"
+COM_USERS_LOGIN_VIEW_DEFAULT_DESC="Displays a login form."
```
Comment (Quy) [inline administrator/language/en-GB/en-GB.com_users.sys.ini:18]: Please sort constant keys in alpha order.

Comment (pe7er) [reply administrator/language/en-GB/en-GB.com_users.sys.ini:18]: mhh, had not thought about that I had to reorder after changing it. Thanks!

**Comment by ReLater** [comment]
It looks like some string changes are missing. Example: See line https://github.com/joomla/joomla-cms/blob/56a0aab1cd15e7afe3cacaf7c0ab5ae179adb98b/components/com_users/tmpl/login/default.xml#L3

**Comment by pe7er** [comment]
> com_user_

Well spotted. I've fixed that... Thanks @ReLater !

**Comment by infograf768** [comment]
Any reason to use lowercase? Looks like this is not normalized in core. Some are lowercase, others are uppercase.

**Comment by ReLater** [comment]
> Any reason to use lowercase?

No, no reason. Just taken over from earlier Joomla versions. All can be uppercase.

**Comment by ReLater** [comment]
I don't have the time to create a pr in this week. Shall I open an issue as reminder?

**Comment by pe7er** [comment]
I'll create a PR for that...

**Comment by pe7er** [comment]
Done! 
PR: https://github.com/joomla/joomla-cms/pull/24815

**Comment by pe7er** [comment]
Closing this issue...
To solve the merge conflicts it was easier to create a new PR for this issue: https://github.com/joomla/joomla-cms/pull/24825

**Pull request closed without merging**